### ISSUE TITLE
Ocean Market Sales Count

### ISSUE THREAD

**Issue by Harky911** [oceanprotocol/market#1740]
In the Ocean market when there is a purchase of a dataset the sales count is not reflected.

**Comment by kremalicious** [comment]
which asset?

**Comment by Harky911** [comment]
Noticed on: Crypto Frequency Analysis

https://market.oceanprotocol.com/asset/did:op:2a664c7701523166c993d6ab8fabdade3eda47d3f171568bcad5e1eed05b75d0

**Comment by Harky911** [comment]
> which dataset are you referring

Noticed on: Crypto Frequency Analysis

https://market.oceanprotocol.com/asset/did:op:2a664c7701523166c993d6ab8fabdade3eda47d3f171568bcad5e1eed05b75d0

**Comment by kremalicious** [comment]
indeed, I get more than 10 orders when [querying subgraph directly](https://v4.subgraph.polygon.oceanprotocol.com/subgraphs/name/oceanprotocol/ocean-subgraph/graphql?query=query+%7B%0A+orders%28where%3A+%7B+datatoken%3A+%220x3be06381775b3aead0ff222c9b1751a9c2f2bf76%22+%7D%29+%7B%0A+++id%0A+%7D%0A%7D). 

The "10 sales" number we are showing on that asset comes from Aquarius response so moving his issue over there.

<img width="488" alt="Screen Shot 2022-10-13 at 00 23 27" src="https://user-images.githubusercontent.com/90316/195465570-cf244299-f266-4e9a-88f0-634fbc2051d1.png">

We ran into issues with Aquarius not picking up things correctly from Polygon assets so this issue here might be caused by this too.
